test/cookbooks/python-webapp-test/metadata.rb

```diff
@@ -0,0 +1,9 @@
+name 'python-webapp-test'
+maintainer 'OSU Open Source Lab'
+maintainer_email 'chef@osuosl.org'
+license 'Apache 2.0'
+description 'Used to test the python-webapp cookbook'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+version '1.0.0'
+
+depends 'python-webapp-cookbook'
```

Comment:
This `depends` is wrong now that you rebased off master. should be `'python-webapp'`